packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-sfb-viewbox-32.tsx

```diff
@@ -14,4 +14,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'sfb-viewbox',
```

Comment:
Can we name the exported icon 'skype' or 'skype-logo'?

Reply:
fixed